src/lib.rs

```diff
@@ -54,8 +54,12 @@ impl Plugin for TilemapPlugin {
         #[cfg(not(feature = "atlas"))]
```

Comment:
Should I just make this `#[cfg(all(not(feature = "atlas"), feature = "render"))]`?

Reply:
Yes